ambry-protocol/src/main/java/com.github.ambry.protocol/PutRequest.java

```diff
@@ -179,20 +200,6 @@ public String toString() {
     return sb.toString();
   }
 
-  // Class to read protocol version 1 Put Request from the stream.
-  private static class PutRequest_V1 {
```

Comment:
No longer used. I just think it is unnecessary to add a comment that says "V1 is no longer used". We can just assume these things.